Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.yml

```diff
@@ -159,6 +159,33 @@ script:
     - contextPath: TAXII.Indicator.Type
       description: The indicator type.
       type: String
+    - contextPath: TAXII.Indicator.title
+      description: The indicator title.
+      type: String
+    - contextPath: TAXII.Indicator.description
+      description: The indicator description.
+      type: String
+    - contextPath: TAXII.Indicator.stixindicatordescription
+      description: The indicator description.
```

Comment:
You have the same descriptions here, same in the "title" output.
Please be more specific so customers will understand the difference.